src/main/java/net/greghaines/jesque/worker/Watchdog.java

```diff
@@ -124,6 +143,40 @@ private void activateIsAliveService(final String serverName) throws RuntimeExcep
         scheduler.scheduleAtFixedRate(isAlive, INITIAL_DELAY, IS_ALIVE_PERIOD, SECONDS);
     }
 
+    private void heartbeatIsAlive(final Jedis jedis) {
+        String recoveringStatus = getRecoveryStatus();
```

Comment:
I don't quite understand why we're passing this to the script.
When you come back - I'll catch to discuss this.

Reply:
Once jesque invokes listener, it needs to know whether the listener is running , finished or failed. This information  is saved in recoveringStatus variable